examples/animation-generic-logo/index.html

```diff
@@ -8,19 +8,35 @@
   </head>
   <body>
     <a-scene>
+      <a-assets>
+        <a-mixin id="frame-edge" geometry="primitive: box; depth: 10; height: 0.1; width: 0.1"
+                                 material="color: #404040"></a-mixin>
+      </a-assets>
 
-      <a-camera fov="80" near="0.1" wasd-controls-enabled="false" cursor-visible="false"></a-camera>
+      <a-camera fov="80" near="0.1" wasd-controls-enabled="false"></a-camera>
 
       <!-- A generic logo made of animated elements -->
-
       <a-entity id="logo" position="0 0.15 -1" scale=".5 .5 .5">
-
         <a-entity id="box">
           <a-animation attribute="scale" from="0 0 0" to="1 1 1" begin="500" dur="500" fill="both" easing="ease-out"></a-animation>
           <a-animation attribute="position" from="0 -0.5 0" to="0 0 0" begin="500" dur="500" fill="both" easing="ease-out"></a-animation>
           <a-animation attribute="rotation" from="0 -300 0" to="30 54 36" begin="500" dur="1200" fill="both" easing="ease-out"></a-animation>
-          <a-cube width="0.25" depth="0.25" height="0.25" color="#EF2D5E"></a-cube>
-          <a-frame scale="0.04 0.04 0.04"></a-frame>
+          <a-box width="0.25" depth="0.25" height="0.25" color="#EF2D5E"></a-box>
+
+          <a-entity scale="0.04 0.04 0.04">
```

Comment:
the "omg it's a-frame in a-frame!" comment? doesn't make much sense now does it :)